.changeset/fluffy-rockets-tickle.md

```diff
@@ -0,0 +1,5 @@
+---
+'@graphprotocol/graph-cli': minor
+---
+
+upgrade to Node 20
```

Comment:
Is there anything needed from the consumer's point of view? Is anything important worth mentioning?